data/keymaps/i386/hcesar/hcesar.map

```diff
@@ -0,0 +1,101 @@
+# hcesar keyboard, for portuguese language
+# based on keyman Hcesar (Portugal) layout
+# Derived from pt-latin1.map 
+# With fixes for keycode 43 from pmmm@camoes.rnl.ist.utl.pt, 980626.
+# With fixes for keycodes 27 and 43 from Francisco Miguel Pedroso Honório Colaço
+# (fcolaco@wolf.fis.uc.pt).
+#
+charset "iso-8859-1"
+keymaps 0-2,4,6,8,12
+alt_is_meta
+include "qwerty-layout"
```

Comment:
Do you need to include qwerty-layout if it's a non-qwerty layout?

Reply:
no